extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-img-quiz.js

```diff
@@ -0,0 +1,133 @@
+
+describes.realWin(
+  'amp-story-interactive-img-quiz',
+  {
+    amp: true,
+  },
+  (env) => {
+    let win;
+    let ampStoryQuiz;
+    let storyEl;
+
+    beforeEach(() => {
+      win = env.win;
+
+      env.sandbox
+        .stub(Services, 'cidForDoc')
+        .resolves({get: () => Promise.resolve('cid')});
+
+      const ampStoryQuizEl = win.document.createElement(
+        'amp-story-interactive-img-quiz'
+      );
+
+      const storeService = new AmpStoryStoreService(win);
+      registerServiceBuilder(win, 'story-store', function () {
+        return storeService;
+      });
+
+      const localizationService = new LocalizationService(win.document.body);
+      env.sandbox
+        .stub(Services, 'localizationServiceForOrNull')
+        .returns(Promise.resolve(localizationService));
+
+      storyEl = win.document.createElement('amp-story');
+      const storyPage = win.document.createElement('amp-story-page');
+      const gridLayer = win.document.createElement('amp-story-grid-layer');
+      gridLayer.appendChild(ampStoryQuizEl);
+      storyPage.appendChild(gridLayer);
+      storyEl.appendChild(storyPage);
+
+      win.document.body.appendChild(storyEl);
+      ampStoryQuiz = new AmpStoryInteractiveImgQuiz(ampStoryQuizEl);
+
+      env.sandbox.stub(ampStoryQuiz, 'mutateElement').callsFake((fn) => fn());
+    });
+
+    it('should create the prompt and options container if there is a prompt', async () => {
+      populateQuiz(ampStoryQuiz, 4, 'Is this a prompt?');
+      await ampStoryQuiz.buildCallback();
+      await ampStoryQuiz.layoutCallback();
+      expect(ampStoryQuiz.getRootElement().children.length).to.equal(2);
+    });
+
+    it('should not create the prompt and options container if there no prompt', async () => {
+      populateQuiz(ampStoryQuiz, 4, undefined);
+      await ampStoryQuiz.buildCallback();
+      await ampStoryQuiz.layoutCallback();
+      expect(ampStoryQuiz.getRootElement().children.length).to.equal(1);
+    });
+
+    it('should structure the content in the quiz element', async () => {
+      populateQuiz(ampStoryQuiz, 4, 'Has prompt!?');
+      await ampStoryQuiz.buildCallback();
+      await ampStoryQuiz.layoutCallback();
+
+      const quizContent = ampStoryQuiz.getRootElement().children;
+      expect(quizContent[0]).to.have.class(
+        'i-amphtml-story-interactive-prompt-container'
+      );
+      expect(quizContent[1]).to.have.class(
+        'i-amphtml-story-interactive-img-option-container'
+      );
+
+      // Check prompt container structure.
+      expect(quizContent[0].children.length).to.equal(1);
+      expect(
+        quizContent[0].querySelectorAll('.i-amphtml-story-interactive-prompt')
+      ).to.have.length(1);
+
+      // Check option container structure.
+      expect(quizContent[1].childNodes.length).to.equal(4);
+      expect(
+        quizContent[1].querySelectorAll(
+          '.i-amphtml-story-interactive-img-option'
+        )
+      ).to.have.length(4);
+    });
```

Comment:
Can you update this test so it checks if the images passed to the options are being used as background for the options?